scss/_variables.scss

```diff
@@ -1526,9 +1533,9 @@ $toast-padding-y:                   $spacer * .25 !default;
 $toast-font-size:                   .875rem !default;
 $toast-color:                       $black !default;
 $toast-background-color:            rgba($white, .85) !default;
-$toast-border-width:                1px !default;
-$toast-border-color:                rgba($black, .1) !default;
 $toast-border-radius:               $border-radius !default;
+$toast-border-width:                var(--#{$variable-prefix}border-width) !default;
```

Comment:
I thought since toasts don't have any design specifications that we could use the same border that Bootstrap for the moment. But I can change it if needed.